docs/pages/build-reference/caching.mdx

```diff
@@ -70,21 +70,40 @@ To disable using our Maven cache server for your builds set `EAS_BUILD_DISABLE_M
 
 ## iOS dependencies
 
-EAS Build runs a CocoaPods cache server that can speed up downloading iOS dependencies for your build jobs. It also makes the service more resilient to CocoaPods CDN outages.
+EAS Build caches the **Podfile.lock** file by default. This provides consistent results across managed app builds.
 
-Currently, EAS Build is configured to cache almost all the pods served from the official CocoaPods CDN, minus a few exceptions which can't be handled by our cache server. These exceptions can be found on the [blacklist](https://github.com/expo/eas-build/blob/main/packages/cocoapods-nexus-plugin/lib/cocoapods_plugin.rb#L4) of our custom CocoaPods plugin and are fetched directly from CDN instead of being fetched through our cache server.
+Additionally, EAS Build runs a CocoaPods cache server that can speed up downloading iOS dependencies for your build jobs. It also makes the service more resilient to CocoaPods CDN outages.
```

Comment:
```suggestion
Additionally, EAS Build runs a CocoaPods cache server that speeds up downloading iOS dependencies for your build jobs. It also makes the service more resilient to CocoaPods CDN outages.
```

Using the word "can" sounds like it may happen or not. If this something being run when creating a build, we can be more affirmative about that the caching will happen? Let me know if I'm understanding it in wrong way.

Reply:
In some cases using the CocoaPods cache server can speed up build time, but in some other cases, it might not happen. I think we are not sure that this will always be true, although it should happen in the majority of cases.

Reply:
If that's the case, maybe we can rephrase it like this:

```
Additionally, EAS Build runs a CocoaPods cache server that can speed up downloading iOS dependencies for your build jobs in most cases.
```

Having that extra context also tells the user that it might not happen in their some cases.